<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qwit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qwit.qrc" >:/images/qwit.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>What are you doing?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="charsLeftLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>140</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>31</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPushButton" >
         <property name="text" >
          <string>Refresh now</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optionsPushButton" >
         <property name="text" >
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>31</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qwit.qrc" />
 </resources>
 <connections/>
</ui>
